### ISSUE TITLE
About  accuracy

### ISSUE THREAD

**Issue by takaniwa** [XuJiacong/PIDNet#70]
Thank you very much for your work on "PIDNet: A Real-time Semantic Segmentation Network Inspired by PID Controllers", I am very impressed. Your explanation of the three modules of PID is brilliant and I have been trying to follow your work recently. However, I'm having trouble reproducing your paper recently, I'm using imagenet pre-trained pidnet-s on four GPUs on the cityspaces training set, and several times I've obtained results around 76%, I'd like to ask you how I can achieve the accuracy that you have in your paper? Thank you very much for your work, looking forward to your reply, sorry if I disturb you!

**Comment by zhujingsong** [comment]
Me too!!! Since I only have one GPU (GTX3070, 8G), I retrained pidnet-s using pidnet_small_cityscapes_trainval.yaml) with batchsize 4. But the results on val set is only around 70%, which is much lower than the reported 78.8%.

I am wondering has anyone has reprodeced the results on Cityscapes???

**Comment by SugarsChieh** [comment]
i can train it to 78.22% in val,but put the result to cityscapes ,i only got 75.4%

**Comment by WenwenZong** [comment]
Hello, my model's training result is 78.3%, but when I generate test results and upload them to the official website of Cityscapes, I only get 76.9%. Even if I use the provided weights, I cannot get the results in the paper. Could you please provide information on how you generated the test set?

**Comment by WenwenZong** [comment]
> i can train it to 78.22% in val,but put the result to cityscapes ,i only got 75.4%

Hello, the same thing is happening to me as well. Have you resolved it yet? If so, could you please share your solution?

**Comment by WenwenZong** [comment]
> Thank you very much for your work on "PIDNet: A Real-time Semantic Segmentation Network Inspired by PID Controllers", I am very impressed. Your explanation of the three modules of PID is brilliant and I have been trying to follow your work recently. However, I'm having trouble reproducing your paper recently, I'm using imagenet pre-trained pidnet-s on four GPUs on the cityspaces training set, and several times I've obtained results around 76%, I'd like to ask you how I can achieve the accuracy that you have in your paper? Thank you very much for your work, looking forward to your reply, sorry if I disturb you!

Hello, the same thing is happening to me as well. Have you resolved it yet? If so, could you please share your solution?
